index.bs

```diff
@@ -476,8 +477,18 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
       <p>
         If present, the <dfn for='VideoConfiguration' dict-member>scalabilityMode</dfn>
         member represents the scalability mode as defined in [[webrtc-svc]].
+        Scalability mode is only applicable to the MediaEncodingType webrtc.
```

Comment:
nits:
* use same scalabilityMode spelling here as in dict-member. 
* make MediaEncodingType and webrtc link to their dfns in this spec.

Reply:
Done.